src/openms/include/OpenMS/FORMAT/FASTAFile.h

```diff
@@ -77,9 +78,9 @@ namespace OpenMS
       String sequence;
 
       FASTAEntry() :
```

Comment:
is FASTAEntry a simple POD? Then we can get rid of most code for construction, destruction and move.
https://stackoverflow.com/questions/146452/what-are-pod-types-in-c?noredirect=1&lq=1
https://stackoverflow.com/questions/12979529/default-constructors-and-pod